.github/workflows/build-on-push.yml

```diff
@@ -43,27 +43,15 @@ jobs:
       # for android, run generate bindings python script
       - name: Generate bindings (Android)
         run: |
-          cd $GITHUB_WORKSPACE/plugin/libs/godot-cpp
-          python ../../../generate.py
+          cd $GITHUB_WORKSPACE/godot-cpp
```

Comment:
You are missing scons, so just add this step infront:
```
      - name: Install scons
        run: |
          python -m pip install scons==4.0.0
```

Reply:
Tried it out yesterday and today, everything from the build step works now, except for uploading the artifact. It can't find the .aar files for some reason, although the path seems correct and an ls can find it.
![workflow_error](https://github.com/user-attachments/assets/128d1fe2-68ee-4ebc-86d6-025f0ec0fbcc)